modules.d/90kernel-modules-extra/module-setup.sh

```diff
@@ -179,8 +180,16 @@ installkernel() {
         | (
             cd "$depmod_module_dir" || exit
             xargs -r realpath -se --
-        ) \
-        | instmods || return 1
+        ) 2>&1 \
+        | while read mod; do
+            if [[ "$mod" == "realpath: "* ]]
+            then
+                prwarning "Non-existing module entry in modules.dep."
```

Comment:
Maybe just instruct to run depmod and re-run dracut and exit the loop.

It might not be enough to just skip the non-existing entries - it might be required to re-run depmod.

Reply:
It is good to instruct to refresh modules.dep. However, I think it is more proper to transfer the option of whether to re-run dracut to the developers.